pyuvdata/uvdata/tests/test_mir_parser.py

```diff
@@ -2274,11 +2274,11 @@ def test_mir_remember_me_sp_data(mir_data):
 
     assert np.all(mir_data.sp_data["iddsmode"] == 0)
 
-    assert np.all(mir_data.sp_data["spareshort"] == 0)
+    assert np.all(mir_data.sp_data["gunnMult"] == 0)
```

Comment:
May be noted above (and if so ignore), but a quick note in the test docstring that this is testing for the existence of the attributes rather than the data types defined in `mir_meta_data.py` would be nice to include.

Reply:
Added on 7cb8ec3d726c5b55599007b7cdc3d465d78ade89.